src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+7.2
+---
+
+ * Deprecate passing a non-empty escape character to the `csv_escape_char` option of `CsvEncoder` and the `CsvEncoder::ESCAPE_CHAR_KEY` constant
```

Comment:
```suggestion
 * Deprecate the `csv_escape_char` context option of `CsvEncoder` and the `CsvEncoder::ESCAPE_CHAR_KEY` constant
```

Same for `UPGRADE-7.2.md`

Reply:
Updated :+1:

Reply:
I think you rolled back these changes in the last commit

Reply:
I don't know what happened, but it should be good now!